tests/integration/crunchers/test_cruncher_quantile_rolling_windows.py

```diff
@@ -133,39 +133,44 @@ def test_relationship_usage(self, simple_df, use_ratio, caplog):
         if use_ratio:
             # We have a 0/0*0 in the calculation, so no value appears.
```

Comment:
no value or zero?

Reply:
An error message appears and 0 is infilled, this is clarified.